src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -315,6 +316,7 @@
       <xsd:element name="service" type="service" />
       <xsd:element name="exclude" type="xsd:string" maxOccurs="unbounded" />
     </xsd:choice>
+    <xsd:attribute name="key-type" type="key_type" default="string" />
```

Comment:
why do we need a default value (above also)?

Reply:
Well I think it answers to https://github.com/symfony/symfony/pull/58035#discussion_r1721889210 :) I removed the default value as well as `string` from the enumeration